traffic_ops/traffic_ops_golang/cdnfederation/cdnfederations.go

```diff
@@ -21,21 +23,27 @@ package cdnfederation
 
 import (
 	"database/sql"
+	_ "embed" // needed to embed SQL queries within Go variables
```

Comment:
None of the file-embedded queries use named parameters. Just use sql, not sqlx, unless I'm missing something?

Reply:
I'm not sure what you mean? `embed` doesn't interact with database drivers or libraries in any way

Reply:
Resolved in 0f6d05dec5